swiftshield-Sources/AutomaticSwiftShield.swift

```diff
@@ -164,6 +164,21 @@ extension AutomaticSwiftShield {
                 guard obfuscationData.usrDict.contains(usr) else {
                     return
                 }
+                
+                if type == .enum && name.hasSuffix("CodingKeys") {
+                    obfuscationData.excludedEnums.insert(usr)
+                    return
```

Comment:
We should do this when we're filling the declarations data, otherwise if the enumelement gets referenced before the declaration they are going to be wrongly obfuscated.

Reply:
OK, I see. I will update it.